src/components/Sidebar/Sidebar.component.jsx

```diff
@@ -1,47 +1,52 @@
 import { React } from "react";
+import { useNavigate } from "react-router-dom";
```

Comment:
The comment i'm gonna leave of fetching the api filtered by category is partially applied here i believe